core/lib/pubs.ts

```diff
@@ -1,16 +1,27 @@
 export interface PubTitleProps {
-	values: { field: { slug: string }; value: unknown }[] | Record<string, unknown>;
+	values:
+		| { field: { slug: string }; value: unknown }[]
+		| Record<string, unknown>
+		| { fieldSlug: string; value: unknown }[];
```

Comment:
🙈 

eventually this should use the new `isTitle` prop but for now, this func just takes a whole bunch of different argument shapes to keep things working without changing too much

Reply:
yeah i was also thinking about how this can be easier, thank you for doing the hard thing for now!

ideally the values from `getPubsWithChildrenetcetc` would also have `isTitle` on them, but i that requires a lot of awkward joins rn